scripts/hyper-pod-template.json

```diff
@@ -14,7 +14,7 @@
         "--",
         "/bin/sh",
         "-c",
-        "while ! [ -f /opt/sd/launch ]; do sleep 1; done; mkdir -p /sd && /opt/sd/launch --api-uri API_URI --store-uri STORE_URI --emitter /opt/sd/emitter BUILD_ID & /opt/sd/logservice --emitter /opt/sd/emitter --api-uri STORE_URI --build BUILD_ID --build-timeout SD_BUILD_TIMEOUT & wait $(jobs -p)"
+        "while ! [ -f /opt/sd/launch ]; do sleep 1; done; mkdir -p /sd && /opt/sd/launch --api-uri API_URI --store-uri STORE_URI --emitter /opt/sd/emitter --build-timeout SD_BUILD_TIMEOUT BUILD_ID & /opt/sd/logservice --emitter /opt/sd/emitter --api-uri STORE_URI --build BUILD_ID & wait $(jobs -p)"
```

Comment:
Shouldn't it be 

`--emitter /opt/sd/emitter BUILD_ID --build-timeout SD_BUILD_TIMEOUT `

Rather than

`--emitter /opt/sd/emitter --build-timeout SD_BUILD_TIMEOUT BUILD_ID`

?

Reply:
Wait, nevermind. LGTM!